drivers/wifi/nrf700x/osal/fw_if/umac_if/src/fmac_api.c

```diff
@@ -4062,6 +4062,17 @@ enum wifi_nrf_status wifi_nrf_fmac_get_power_save_info(void *dev_ctx,
 
 	return status;
 }
+
+#ifdef CONFIG_NRF_WIFI_LOW_POWER
+enum wifi_nrf_rpu_ps_state wifi_nrf_fmac_get_rpu_ps_state(void *dev_ctx)
```

Comment:
The naming is misleading, this is not a `current` PS state of RPU, this is just host's perspective, RPU could still be dozing or checking beacon. So, we should name it appropriately.